### PR TITLE
audio demo: add bitrate graph

### PR DESCRIPTION
**Pull request by fippo**
This adds a bitrate display to the audio-only demo. This is actually pretty interesting as it allows to visually  compare the audio bitrates of several codecs and the effects of silence.

The graph code is taken from webrtc-internals (removing the use of $ in two places and lots  of changes to make jscs and jshint happy) so that should not be a license problem. It doesn't really look as pretty as highcharts but it's ok.
Maybe it should go to src/js/lib so it can be reused by other demos?

@juberti @samdutton @jiayliu

### CODE REVIEW

**Comment by samdutton** [comment]
Looks good – minor detail: could you sort out the code formatting problem that's causing the Travis failure? Thanks.

**Comment by fippo** [comment]
done, should also fix the issue in #496

**Comment by fippo** [comment]
disabling VAD suggested by @juberti -- this moves the PCMU bitrate back to where I would expect it.

Do we want this to be a "disable VAD" checkbox that is on by default? I'd really like to see a blog post explaining this.

File: src/content/peerconnection/audio/js/graph.js [hunk 1]
```diff
@@ -0,0 +1,690 @@
+// Copyright (c) 2013 The Chromium Authors. All rights reserved.
```
Comment (juberti) [inline src/content/peerconnection/audio/js/graph.js]: Sam, how should we deal with this copyright, which is slightly different than the webrtc-samples copyright?

**Comment by fippo** [comment]
Reduced canvas size to the same size used in webrtc-internals, this looked blurry.
Also added a graph for the number of packets which should be interesting when ptime can be tweaked.

**Comment by KaptenJansson** [comment]
Ping samdutton@ regarding juberti@'s comment about copyright.

**Comment by samdutton** [comment]
Oops – sorry. I looked into this, but forgot to follow it up. Just checking
now.

**Comment by samdutton** [comment]
From the open-sourcing team at Google: 'please use the same WebRTC project authors heading throughout. If you need to, you can amend the AUTHORS files to also list "The Chromium Authors".'

In general they recommend putting all code like this in a third_party folder.

**Comment by KaptenJansson** [comment]
@fippo ping regarding @samdutton's last comment.

**Comment by fippo** [comment]
@samdutton how is that?

**Comment by samdutton** [comment]
I followed up on this with open source team here (but forgot to update
here!)

They say:

No [don't use two licence notices], please use the same WebRTC project
> authors heading throughout. If you need to, you can amend the AUTHORS files
> to also list "The Chromium Authors".

**Comment by fippo** [comment]
assuming that there will be an AUTHORS file after merge :-)

**Comment by samdutton** [comment]
@fippo please sanity check the AUTHORS file I just added :).

**Comment by fippo** [comment]
LGTM. I assume that me and @jan-ivar count as "webrtc project authors" since we signed the CLA (and heck, we have https://github.com/webrtc/samples/graphs/contributors anyway)

**Comment by samdutton** [comment]
@juberti

Is 'WebRTC Project Authors' equivalent to repo contributors?

**Comment by KaptenJansson** [comment]
I cannot find any other suitable "author" group unless we make up a new one like "WebRTC Project Contributors" which I think it's too close to "WebRTC Project Authors" to bother. But I'm no expert in that area.

**Comment by KaptenJansson** [comment]
Ping, I would like to merge this.

**Comment by samdutton** [comment]
I think we should stick with 'WebRTC Project Authors' – can change if
necessary.

**Pull request merged**